spec/requests/as_server_app/comment_thread_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+require 'yt/models/comment_thread'
+
+describe Yt::CommentThread, :server_app do
+  subject(:comment_thread) { Yt::CommentThread.new attrs }
+
+  context 'given an existing comment thread ID about a channel' do
+    let(:attrs) { {id: 'z13vsnnbwtv4sbnug232erczcmi3wzaug'} }
+
+    it { expect(comment_thread.video_id).to be_nil }
+    # Not sure why but channel_id seems always missing value
+    it { expect(comment_thread.channel_id).to be_nil }
+    it { expect(comment_thread.total_reply_count).to be_an Integer }
+    it { expect(comment_thread.can_reply).to be_boolean }
+    it { expect(comment_thread.is_public).to be_boolean }
```

Comment:
Once you have _real_ booleans, you can write tests like this:

https://github.com/Fullscreen/yt/blob/master/spec/models/resource_spec.rb#L22-L26